lib/Common/ConfigFlagsList.h

```diff
@@ -623,6 +623,7 @@ PHASE(All)
 #define DEFAULT_CONFIG_ES6Spread               (true)
 #define DEFAULT_CONFIG_ES6String               (true)
 #define DEFAULT_CONFIG_ES6StringPrototypeFixes (true)
+#define DEFAULT_CONFIG_ES2019ObjectSpread       (true)
```

Comment:
Perhaps you could change this to default to `false`, so you can get this checked-in before moving on to supporting it in the JIT? Or if you want to wait until you have the whole thing done, that's fine too.

Reply:
good idea!